bosh-upload-release/task

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eux
```

Comment:
We're trying to move away from using `-x` to make the output more readable/intentional/secure. Could you please remove it from the script?

Reply:
Will do

Reply:
It looks like you removed the `-e` flag (fast fail if a variable does not exist) by mistake instead of `-x` (print each command before executing it). Could you please fix this?

Reply:
oops